test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -44,8 +44,10 @@ Describe "PSReadLine" -tags "CI" {
 
     It "Should ignore case when using Function binding" {
         Set-PSReadlineOption -EditMode Windows
-        Set-PSReadlineKeyHandler "Ctrl+k" -Function yank # NOTE: lower case function name
-        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be Yank
+        $lowerCaseFunctionName = "yank";
+        $pascalCaseFunctionName = "Yank"
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function $lowerCaseFunctionName
+        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be $pascalCaseFunctionName
```

Comment:
Looking at this now, having a variable named `$pascalCaseFunctionName` seems to be implying some fragility here when there is none. I think this really should be a "Yank" string literal. I looked at the code and whatever was used for the binding is assigned to the Function property, so in this case, the output is actually the same as the input, so the comparison should be against `$lowerCaseFunctionName` lol... I'll push another change. Sigh.